fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java

```diff
@@ -75,15 +76,19 @@ public static void init() throws UnknownHostException {
                 } else if (!priorityCidrs.isEmpty()) {
                     if (isInPriorNetwork(addr.getHostAddress())) {
                         localAddr = addr;
+                        checkIpFlag = true;
                         break;
-                    }
+                    } 
                 } else {
                     localAddr = addr;
                     break;
                 }
             }
         }
-
+        //if all ips not match the priority_networks then print the warning log
+        if (!checkIpFlag) {
```

Comment:
When priority_network is not set, checkIpFlag is false, this branch will execute. That is not the actual intent.
So I think priorityCidrs.isEmpty() should also be checked here.

About the flag name, I prefer hasMatchedIp.

Reply:
Okay, let me modify my code